src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/_params.py

```diff
@@ -3,116 +3,81 @@
 # --------------------------------------------------------------------------------------------
 
-from azure.cli.command_modules.lab.validators import (validate_lab_vm_create,
-                                                      validate_lab_vm_list,
-                                                      validate_user_name,
-                                                      validate_template_id,
-                                                      validate_claim_vm,
-                                                      _validate_artifacts)
-from azure.cli.core.commands.parameters import resource_group_name_type
-from azure.cli.core.commands import ParametersContext
+from azure.cli.command_modules.lab.validators import validate_artifacts, validate_template_id
 from azure.cli.core.util import get_json_object
 
 
-with ParametersContext(command='lab') as c:
-    c.argument('resource_group', arg_type=resource_group_name_type,
-               help='Name of lab\'s resource group. You can configure the default group '
-                    'using \'az configure --defaults group=<name>\'')
+def load_arguments(self, _):
+    with self.argument_context('lab custom-image create') as c:
+        c.argument('name', options_list=['--name', '-n'])
 
-with ParametersContext(command='lab custom-image create') as c:
-    c.register_alias('resource_group', ('--resource-group', '-g'))
-    c.register_alias('name', ('--name', '-n'))
+    with self.argument_context('lab vm create') as c:
+        c.argument('name', options_list=['--name', '-n'])
 
-with ParametersContext(command='lab vm create') as c:
-    c.register_alias('resource_group', ('--resource-group', '-g'), validator=validate_lab_vm_create)
-    c.register_alias('name', ('--name', '-n'))
+        # Authentication related arguments
+        for arg_name in ['admin_username', 'admin_password', 'authentication_type', 'ssh_key', 'generate_ssh_keys',
+                         'saved_secret']:
+            c.argument(arg_name, arg_group='Authentication')
+        c.argument('generate_ssh_keys', action='store_true')
 
-    # Authentication related arguments
-    authentication_group_name = 'Authentication'
-    c.argument('admin_username', arg_group=authentication_group_name)
-    c.argument('admin_password', arg_group=authentication_group_name)
-    c.argument('authentication_type', arg_group=authentication_group_name)
-    c.argument('ssh_key', arg_group=authentication_group_name)
-    c.argument('generate_ssh_keys', action='store_true', arg_group=authentication_group_name)
-    c.argument('saved_secret', arg_group=authentication_group_name)
+        # Add Artifacts from json object
+        c.argument('artifacts', type=get_json_object)
 
-    # Add Artifacts from json object
-    c.argument('artifacts', type=get_json_object)
+        # Image related arguments
+        for arg_name in ['os_type', 'gallery_image_reference', 'custom_image_id']:
+            c.ignore(arg_name)
```

Comment:
Please replace this with:
`c.ignore('os_type', 'gallery_image_reference', 'custom_image_id')`